pkgs/development/compilers/openjdk/openjfx/17.nix

```diff
@@ -14,6 +14,16 @@ let
     java = openjdk17_headless;
   });
 
+  icu-data = fetchurl {
+    url = "https://github.com/unicode-org/icu/releases/download/release-71-1/icu4c-71_1-data-bin-l.zip";
+    hash = "sha256-pVWIy0BkICsthA5mxhR9SJQHleMNnaEcGl/AaLi5qZM=";
+  };
+
+  fakeRepository = runCommand "icu-data-repository" {} ''
+    mkdir -p $out/download/release-71-1
+    cp ${icu-data} $out/download/release-71-1/icu4c-71_1-data-bin-l.zip
+  '';
```

Comment:
We should assert that the ico version matches

Reply:
I used the version from the icu package to build the fake repository. This way the versions have to match.